about.html

```diff
@@ -13,6 +13,11 @@
     <link rel="stylesheet" href="styles/light_dark.css">
     <script src="Js/modal_header.js" defer></script>
     <script src="Js/light_dark.js" defer></script>
+
+    <div class="gtranslate_wrapper"></div>
+    <script>window.gtranslateSettings = { "default_language": "en", "detect_browser_language": true, "languages": ["en", "pt", "fr", "es"], "wrapper_selector": ".gtranslate_wrapper", "flag_style": "3d" }</script>
+    <script src="https://cdn.gtranslate.net/widgets/latest/float.js" defer></script>
+
```

Comment:
Vou adicionar as sugestões dai vc vai aplicando tá?
```suggestion
<script src="Js/translations.js" defer></script>
```

Reply:
ok